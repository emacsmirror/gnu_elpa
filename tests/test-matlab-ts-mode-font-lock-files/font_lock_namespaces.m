% -*- matlab-ts -*-

% Structure, bar and disp are valid fields and shouldn't have builtin face.
foo.bar.disp.goo = 1;

%  Case: "Simulink.MDLInfo.FileName" 'property
x = Simulink.MDLInfo.FileName;

%  Case: "Simulink.CMI.CompiledSimType.ModelApi" 'enumeration
y = Simulink.CMI.CompiledSimType.ModelApi;

% genapp is a function in the simulink.compiler namespace
simulink.compiler.genapp('modelName', foo)

% test with no args (though this will error when run)
simulink.compiler.genapp()

% test again with no args (though this will error when run)
simulink.compiler.genapp
