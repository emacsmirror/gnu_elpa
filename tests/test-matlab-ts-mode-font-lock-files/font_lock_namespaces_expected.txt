c ccc ccccccccc ccc

c cccccccccc ccc ccc cccc ccc ccccc cccccc ccc ccccccccc cccc ccccccc ccccc
dddDdddDddddDddd o nD

c  ccccc ccccccccccccccccccccccccccc ccccccccc
v o BBBBBBBBBBBBBBBdDPPPPPPPPD

c  ccccc ccccccccccccccccccccccccccccccccccccccc cccccccccccc
v o BBBBBBBBBBBBBBBBBBBBBBBBBBBdDPPPPPPPPD

c cccccc cc c cccccccc cc ccc ccccccccccccccccc ccccccccc
BBBBBBBBBBBBBBBBBBBBBBBBbSsssssssssSD dddb

c cccc cccc cc cccc ccccccc cccc cccc ccccc cccc cccc
BBBBBBBBBBBBBBBBBBBBBBBBbb

c cccc ccccc cccc cc cccc ccccccc cccc cccc ccccc cccc cccc
BBBBBBBBBBBBBBBBBBBBBBBB
